requirements.txt

```
@@ -2,4 +2,4 @@ certifi>=2018.4.16
 expiringdict>=1.1.4
 pyRFC3339>=1.0
 semver>=2.10.2,<3.0.0
-urllib3>=1.22.0
+urllib3>=1.22.0,<1.27
```

I know this isn't likely to happen, but a 2.0 release doesn't mean there couldn't potentially be another 1.x release so I think if we move forward with this PR (see comments on the issue you reported), I would prefer this to be updated as follows:

```suggestion
urllib3>=1.22.0,<2.0.0
```